pyresample/spherical_geometry.py

```diff
@@ -236,17 +236,23 @@ def angle(self, other_arc, snap=True):
 
     def intersections(self, other_arc):
         """Get the two intersections of the greats circles defined by the current arc and *other_arc*."""
+        end_lon = self.end.lon
```

Comment:
This function and the one in spherical.py are really similar, could the common code be factorized?

Reply:
A sort of. Coordinate can accept more types of args (in an unsafe way): 
The problem is rather that `spherical_geometry.Arc` methods seem to not work really well. 
There is also a test failing ...

Reply:
Ok, I see now that we have marked spherical_geometry.py for deprecation in pyresample 2.0. I suggest we leave that module as it is, and focus on spherical.py only.

Reply:
As far as deprecation, @ghiggi has pointed out in other threads that it is still heavily used in the geometry classes. We'll need to work on that.

Reply:
I guess I was hoping the updates to the geometry classes would be simple like changing the import and the class name used.